libs/utils/include/utils/compiler.h

```diff
@@ -189,7 +189,7 @@
 
 
 // ssize_t is a POSIX type.
-#if defined(WIN32)
```

Comment:
There are a lot more places where we use `WIN32`, this should at least be `#if defined(WIN32) || defined(_WIN32)`

Reply:
Yeah i changed it to WIN32 or _WIN32.